opencensus/proto/stats/metrics/metrics.proto

```diff
@@ -69,15 +69,20 @@ message MetricDescriptor {
   Type type = 4;
 
   // TODO: Consider to add LabelInfo{key, description} if needed.
+  // The label keys associated with the metric descriptor. Label keys must be
+  // unique and in sorted order.
+  repeated string label_keys = 5;
 }
 
 // A collection of data points that describes the time-varying values
 // of a metric.
 message TimeSeries {
   // TODO: Add restrictions for characters that can be used for keys and values.
+  // TODO: Wrap values to distinguish between empty-string and not-set.
   // The set of label values that uniquely identify this timeseries. Apply to all
-  // points.
-  map<string, string> label = 1;
+  // points. The order of label values must match that of label keys in the
+  // metric descriptor.
+  repeated string label_values = 1;
```

Comment:
Use google.protobuf.StringValue see for example:
https://github.com/census-instrumentation/opencensus-proto/blob/master/opencensus/proto/trace/trace.proto#L247

You need to add `import "google/protobuf/wrappers.proto";`

Reply:
Done.